src/components/Organization/ViewOrganization/index.js

```diff
@@ -48,12 +48,30 @@ const ViewOrganization = () => {
   const [people, setPeople] = useState([]);
   const [orgFollowed, setOrgFollowed] = useState([]);
   const [Activity, setActivity] = useState(1);
+
+  // Firebase Hooks
   const firebase = useFirebase();
   const dispatch = useDispatch();
   const firestore = useFirestore();
+
   const db = firebase.firestore();
   const profileData = useSelector(({ firebase: { profile } }) => profile);
+  // console.log(firebase);
+  const CurrentOrg = useSelector(
+    ({
+      profile: {
+        data: { organizations }
+      },
+      org: {
+        general: { current }
+      }
+    }) => organizations[0]
+  );
 
+  console.log(CurrentOrg);
```

Comment:
remove this console.log

Reply:
check other files too, they have console.log functions as well

Reply:
done👍